.buildkite/deploy/kubernetes/scripts/setup-k8s-env.sh

```diff
@@ -5,10 +5,10 @@ set -euo pipefail
 source .buildkite/env-scripts/util.sh
```

Comment:
is this still needed?

Reply:
Yes, to use `add_bin_path`. Sourcing same method from `common.sh` breaks retry (overrides the general retry command).